operator/src/main/resources/scripts/introspectDomain.sh

```diff
@@ -61,21 +68,68 @@ function createFolder {
   fi
 }
 
-trace "Introspecting the domain"
+#
+# setup MII functions in case this is a MII domain
+#
+
+source ${SCRIPTPATH}/modelInImage.sh
+[ $? -ne 0 ] && trace SEVERE "Error sourcing ${SCRIPTPATH}/modelInImage.sh" && exit 1
+
+#
+# setup introspector log file
+#
+
+traceDirs before LOG_HOME
+
+if [ ! -z "${LOG_HOME}" ] && [ ! -d "${LOG_HOME}" ]; then
+  trace "Creating log home directory: '${LOG_HOME}'"
+  createFolder ${LOG_HOME}
+fi
+
+ilog_dir="${LOG_HOME:-/tmp}"
+ilog_timestamp="$(date --utc '+%Y-%m-%d_%H.%M.%S')"
+ilog_prefix="${ilog_dir}/introspector_script"
+ilog_file="${ilog_prefix}_${ilog_timestamp}_.out"
+ilog_max=${MAX_INTROSPECTOR_LOG_FILES:-10}
+
```

Comment:
Implementation note: MAX_INTROSPECTOR_LOG_FILES is internal for now - there are no plans to document this env var.  Default is 10.